src/decorator-annotator.ts

```diff
@@ -51,7 +51,7 @@ class ClassRewriter extends Rewriter {
   }
 
   private decoratorsToLower(n: ts.Node): ts.Decorator[] {
-    return (n.decorators || []).filter(d => this.shouldLower(d));
+    return (n.decorators as ts.NodeArray<ts.Decorator>|| []).filter(d => this.shouldLower(d));
```

Comment:
I think `node.decorators` ends up as some odd union type because it's defined multiple times in the dependency hierarchy of `ts.Node`. No idea why it happens in this version though.